armi/nuclearDataIO/cccc/tests/test_dif3d.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2019 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test reading/writing of DIF3D binary input."""
+
+import os
+import unittest
+
+from armi.nuclearDataIO.cccc import dif3d
+from armi.utils.directoryChangers import TemporaryDirectoryChanger
+
+THIS_DIR = os.path.dirname(__file__)
+
+SIMPLE_HEXZ_INP = os.path.join(THIS_DIR, "../../tests", "simple_hexz.inp")
+SIMPLE_HEXZ_DIF3D = os.path.join(THIS_DIR, "fixtures", "simple_hexz.dif3d")
+
+
+class TestDif3d(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Load DIF3D data from binary file."""
```

Comment:
Consider adding a sentence or two on how this file was generated and perhaps what version of DIF3D you used to make it.

Not sure the best place to do this, so perhaps here?

Reply:
nice suggestion. done!